server/mail/accountVerification/verificationEmailTemplate.js

```diff
@@ -0,0 +1,32 @@
+import Mailgen from "mailgen";
```

Comment:
Change this folder structure to `server/emails/verification`
- One file is `email`
- The other is the `config` for the email